src/themes/components/card.ts

```diff
@@ -0,0 +1,30 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+
+const parts = ['carTitle', 'price', 'dealerName', 'dealerAddress'];
+
+const Card: ComponentStyleConfig = {
+  parts,
+  baseStyle: {
+    carTitle: {
+      color: 'gray.900',
+      textStyle: { xs: 'heading5', lg: 'heading3' },
+      noOfLines: { xs: 1, lg: 'none' },
+    },
+    price: {
+      color: 'gray.900',
+      textStyle: { xs: 'heading3', lg: 'lg' },
+    },
+    dealerName: {
+      color: 'gray.900',
+      textStyle: 'heading4',
+      display: { xs: 'none', lg: 'flex' },
+    },
+    dealerAddress: {
+      color: 'gray.900',
+      textStyle: 'body',
```

Comment:
For all the texStlye I'm still waiting for Enrique's answer because the styling values in figma are not in sync with the token we have.